go/pkg/client/cas.go

```diff
@@ -1377,3 +1387,14 @@ func (c *Client) DownloadFiles(ctx context.Context, execRoot string, outputs map
 	}
 	return nil
 }
+
+func (c *Client) shouldCompress(sizeBytes int64) bool {
```

Comment:
nit: If the function is not reused, maybe inline?

Reply:
I'll be reusing it on read compression :)